app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+  def create
+  	user = User.find_by(email: params[:emails]["0"][:value])
+  	if !user
+  		# Strip trailing "?sz=50" from Google Plus image URL
+  		image_url = params[:image][:url]
+  		image_url = image_url[0..image_url.index('?') - 1]
+
+  		user = User.create(first_name:params[:name][:givenName], 
+  						last_name:params[:name][:familyName], 
+  						email:params[:emails]["0"][:value], 
+  						image:image_url)
```

Comment:
standard ruby style is to have a single whitespace after a hash key, so `first_name: params[...]...`.

Also, all this functionality should either go in a private helper method or in a service